content/_includes/page/intro.njk

```diff
@@ -0,0 +1,13 @@
+{% import 'utility.macros.njk' as utility %}
+
+{% if summary or action %}
+  <div id='intro'>
```

Comment:
should we use double quotes in html? I see it as double in other files when using straight up html.